docs/docs/development/database/migration-plan.md

```
@@ -0,0 +1,979 @@
+
+   ```prisma
+   model User {
+     // 既存フィールド
+     // ...
+
+     workTasks WorkTask[]
+   }
+   ```
+
+3. **マイグレーション実行**
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)